docs/zh-CN/docs/sql-manual/sql-functions/table-functions/s3.md

```diff
@@ -80,6 +80,11 @@ S3 tvf中的每一个参数都是一个 `"key"="value"` 对。
 - `num_as_string`： (选填) 默认为 `false`
 - `fuzzy_parse`： (选填) 默认为 `false`
 
+    下面2个参数是用于csv格式的导入
```

Comment:
you need to wrap this with `<version since='dev'>` tag. See:
https://doris.apache.org/zh-CN/community/how-to-contribute/contribute-doc/#%E6%96%87%E6%A1%A3%E5%A4%9A%E7%89%88%E6%9C%AC

Reply:
done